subprojects/docs/src/docs/userguide/dependencyManagementForJavaProjects.adoc

```diff
@@ -0,0 +1,109 @@
+// Copyright 2017 the original author or authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+[[dependency_management_for_java_projects]]
+== Dependency Management for Java Projects
+
+This chapter explains how to apply basic dependency management concepts to Java-based projects. For a detailed introduction to dependency management, see <<introduction_dependency_management>>.
+
+[[sec:setting_up_a_standard_build_script_java_tutorial]]
+=== Dissecting a typical build script
+
+Let's have a look at a very simple build script for a Java-based project. It applies the <<java_library_plugin,Java Library plugin>> which automatically introduces a standard project layout, provides tasks for performing typical work and adequate support for dependency management.
+
+++++
+<sample id="basicDependencyDeclarations" dir="userguide/artifacts/dependencyBasics" title="Dependency declarations for Java-based project">
```

Comment:
"Dependency declarations for Java-based project" 
-> 
"Dependency declarations for **a** Java-based project"

Reply:
done